### PR TITLE
Add Data Type column in METRICS.md

### LINKED ISSUE #6: "Data type" column in Collected metrics table
Could you provide "data type" column for collected metrics table in METRICS.md like [here](https://github.com/intelsdi-x/snap-plugin-collector-swap/blob/master/METRICS.md).

### PR DESCRIPTION
**Pull request by kdembler**
Resolves #6

### CODE REVIEW

File: METRICS.md [hunk 1]
```diff
@@ -3,26 +3,26 @@
...
+----------|-----------|-------------
+/intel/yarn/queue/[queue_id]/absolutecapacity | float64 | Absolute capacity percentage this queue can use of entire cluster
+/intel/yarn/queue/[queue_id]/absolutemaxcapacity | int | Absolute maximum capacity percentage this queue can use of the entire cluster
+/intel/yarn/queue/[queue_id]/absoluteusedcapacity | int | Absolute used capacity percentage this queue is using of the entire cluster
+/intel/yarn/queue/[queue_id]/capacity | int | Configured queue capacity in percentage relative to its parent queue
+/intel/yarn/queue/[queue_id]/maxactiveapplications | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxactiveapplicationsperuser | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxapplications | int | The maximum number of active applications this queue can have
+/intel/yarn/queue/[queue_id]/maxapplicationsperuser | int | The maximum number of active applications per user this queue can have
+/intel/yarn/queue/[queue_id]/maxcapacity | int | Configured maximum queue capacity in percentage relative to its parent queue
```
Comment (IzabellaRaulin) [inline METRICS.md]: Are you sure this is int? Please, look on code or even o this example of response - https://github.com/intelsdi-x/snap-plugin-collector-yarn/blob/master/yarncollector/response.json#L102

File: METRICS.md [hunk 2]
```diff
@@ -3,26 +3,26 @@
...
-/intel/yarn/queue/*/userlimitfactor | The user limit factor set in the configuration
-/intel/yarn/queue/*/resources_memory | The amount of memory used (in MB)
-/intel/yarn/queue/*/resources_vcores | The number of virtual cores
-/intel/yarn/schedulerinfo/capacity | Configured queue capacity in percentage for root queue
-/intel/yarn/schedulerinfo/maxcapacity | Configured maximum queue capacity in percentage for root queue
+Namespace | Data Type | Description
+----------|-----------|-------------
+/intel/yarn/queue/[queue_id]/absolutecapacity | float64 | Absolute capacity percentage this queue can use of entire cluster
+/intel/yarn/queue/[queue_id]/absolutemaxcapacity | int | Absolute maximum capacity percentage this queue can use of the entire cluster
+/intel/yarn/queue/[queue_id]/absoluteusedcapacity | int | Absolute used capacity percentage this queue is using of the entire cluster
```
Comment (IzabellaRaulin) [inline METRICS.md]: Are sure that is int data type?

File: METRICS.md (hunk 1 above)
Comment (IzabellaRaulin) [reply METRICS.md]: @kdembler, I could you check if declared data types for each of listed metrics is correct?

Comment (kdembler) [reply METRICS.md]: https://github.com/intelsdi-x/snap-plugin-collector-yarn/blob/master/yarn/xml.go#L57 Apparently even though JSON response contains data that looks like `float`, plugin parses it to `int`, and this is the type that is exposed.

File: METRICS.md (hunk 2 above)
Comment (kdembler) [reply METRICS.md]: https://github.com/intelsdi-x/snap-plugin-collector-yarn/blob/f68084d97fea4ce96ef6eccd5341ea0ad6a6b269/yarn/xml.go#L51
Sure looks like it :)

File: METRICS.md (hunk 1 above)
Comment (IzabellaRaulin) [reply METRICS.md]: ok, so then it looks good to me

**Pull request merged**